src/main/java/uk/gov/companieshouse/api/accounts/AccountsType.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.companieshouse.api.accounts;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * An enumeration of account types
+ */
+public enum AccountsType {
+
+    SMALL_FULL_ACCOUNTS("small-full", "small-full-accounts.html", "accounts#smallfull", "small_full_accounts", "small-full-accounts"),
+    ABRIDGED_ACCOUNTS("abridged", "abridged-accounts.html", "accounts#abridged", "abridged_accounts", "abridged-accounts"),
+    DORMANT_ACCOUNTS("dormant", null, "accounts#dormant", "dormant_accounts", "dormant-accounts"),
```

Comment:
Should we treat `null` as a dormant account?

Reply:
This was copied from abridged and i didnt want to delete the other accounts type to have them as a reference; but i think i will delete them so that it wont cause any confusion.